web/app/scripts/src/providers/project-config.es.js

```diff
@@ -0,0 +1,22 @@
+const ProjectConfig = function () {
+    let settings = {
+        apiRoot: 'localhost:8000'
+    };
+
+    return {
+        $get: function () {
+            return {
+                getApiRoot: () => settings.apiRoot
+            }
+        },
+        setApiRoot: (value) => {
+            settings.apiRoot = value;
+        }
+    };
+}
+
+export const provider = [
```

Comment:
Yeah I was thinking about it, should the dependencies be supplied in the `export` or when you register the thing in the main module. There's pros and cons for both, the angular dependency injector is not really suited to work well with ES6 modules